Open_Foam/Domain.ipynb

```diff
@@ -52,23 +45,29 @@
     "    (0.5 0 1)\n",
     "    (32 0 1)\n",
     ");\n",
-    "</pre>\n",
+    "```\n",
     "\n",
-    "As you may have mentioned, vertices are defined on two parallel planes along the z direction. In OpenFOAM there are no surface-type meshes. All meshes are in 3D, instead in 2D problems, the boundaries in the normal direction are defined as empty type. You can find better explanation on this type of boundary condition in [here](http://cfd.direct/openfoam/user-guide/boundaries/).\n",
+    "It is interesting to note that looking at these coordinates of these vertices reveals the fact that all of them are defined on two parallel xy planes with unit length offset. The reason behind this is that in OpenFOAM the concept of surface meshe is not defined! All mesh elements should be defined in 3D space. To model 2D case studies the `type` for boundaries in the third dimensions are defined as `empty`. More information and detail explanation on boundary type can be read [here](http://cfd.direct/openfoam/user-guide/boundaries/).\n",
     "\n",
-    "The vertices are automatically labeled by blockMesh by their order. The first vertex would be labeled as 0, the second vertex would be 1 and this numbering process continues up to the last vertex. The next step would be defining the two blocks based on the vertices. This is done by calling the vertices by their respective labels.\n",
+    "The above-defined vertices are automatically labeled numerically by `blockMesh` in the order that are input into the `blockMesh` file. First input vertex would be labeled with number 0, the second vertex would be labeled with number 1 and this labeling process continues up to the last input vertex. These labeling process is visualized in Fig. 3.\n",
     "\n",
```

Comment:
We should replace this figure (Fig. 3) with a figure that only visualizes the vertices with their corresponding labels. We are going to discuss the edges creation in the next step, thus there is no point to visualize them here. It can be confusing.